### PR TITLE
EPMRPP-33157: filter modals

### PR DESCRIPTION
**Pull request by DiscoElevator**

### CODE REVIEW

File: app/localization/translated/be.json [hunk 1]
```diff
@@ -29,6 +39,7 @@
   "FilterTableItem.shared": "Апублікаваны:",
   "FilterTableItem.showOnLaunchesSwitcherOff": "ВЫКЛ",
   "FilterTableItem.showOnLaunchesSwitcherOn": "УКЛ",
+  "Filters.share": "Share",
```
Comment (ArtsiomYeliseyenka) [inline app/localization/translated/be.json]: translate please

File: app/localization/translated/be.json [hunk 2]
```diff
@@ -37,6 +48,7 @@
   "ForgotPasswordForm.cancel": "Адмяніць",
   "ForgotPasswordForm.emailPlaceholder": "Увядзіце электронны адрас",
   "ForgotPasswordForm.sendEmail": "Адаслаць ліст",
+  "Input.switcher": "ON",
```
Comment (ArtsiomYeliseyenka) [inline app/localization/translated/be.json]: translate please

File: app/localization/translated/ru.json [hunk 3]
```diff
@@ -29,6 +39,7 @@
   "FilterTableItem.shared": "Поделиться:",
   "FilterTableItem.showOnLaunchesSwitcherOff": "Выкл",
   "FilterTableItem.showOnLaunchesSwitcherOn": "Вкл",
+  "Filters.share": "Share",
```
Comment (ArtsiomYeliseyenka) [inline app/localization/translated/ru.json]: translate please

File: app/localization/translated/ru.json [hunk 4]
```diff
@@ -37,6 +48,7 @@
   "ForgotPasswordForm.cancel": "Отменить",
   "ForgotPasswordForm.emailPlaceholder": "Введите email",
   "ForgotPasswordForm.sendEmail": "Отправить письмо",
+  "Input.switcher": "ON",
```
Comment (ArtsiomYeliseyenka) [inline app/localization/translated/ru.json]: translate please

File: app/src/pages/inside/filtersPage/filterDeleteModal/filterDeleteModal.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  deleteFilterText: {
+    id: 'DeleteFilterDialog.deleteFilter',
+    defaultMessage: 'Are you sure to delete filter \'{name}\'? It will no longer exist.',
+  },
+});
+
+@withModal('filterDeleteModal')
+@injectIntl
+export class FilterDeleteModal extends Component {
+  static propTypes = {
```
Comment (SergeyKovalchuk) [inline app/src/pages/inside/filtersPage/filterDeleteModal/filterDeleteModal.jsx]: Please use common way to build prop types and default props

File: app/src/pages/inside/filtersPage/filterDeleteModal/filterDeleteModal.jsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  static defaultProps = {
+    data: {
+      filter: {},
+      onConfirm: () => {
+      },
+    },
+  };
+
+  render() {
+    const { intl } = this.props;
```
Comment (SergeyKovalchuk) [inline app/src/pages/inside/filtersPage/filterDeleteModal/filterDeleteModal.jsx]: intl?

Comment (ArtsiomYeliseyenka) [reply app/src/pages/inside/filtersPage/filterDeleteModal/filterDeleteModal.jsx]: https://github.com/yahoo/react-intl

**Pull request merged**